schemas/JSON/manifests/v1.2.0/manifest.defaultLocale.1.2.0.json

```
@@ -154,6 +169,23 @@
       "$ref": "#/definitions/Url",
       "description": "The package release notes url"
     },
+    "PurchaseUrl": {
+      "$ref": "#/definitions/Url",
+      "description": "The purchase url for acquiring entitlement for the package."
+    },
+    "InstallationNotes": {
+      "type": [ "string", "null" ],
+      "minLength": 1,
+      "maxLength": 240,
+      "description": "The notes displayed to the user upon completion of a package installation."      
+    },
+    "Documentations": {
+      "type": [ "array", "null" ],
+      "items": {
+        "$ref": "#/definitions/Documentation"
+      },
+      "maxItems": 128

```

```suggestion
      "maxItems": 256
```
I know that this may seem a bit overkill, but given that we have seen issues created to increase the number of installer nodes and file extensions, I think making this larger off the start may be a wise choice

---

Updated maxItems from 128 to 256